test/distributed/test_c10d_common.py

```diff
@@ -1675,6 +1675,15 @@ def comm_fn(tensor, group=None):
 
         self._test_work_wait(tensor, comm_fn=comm_fn)
 
+    def _test_reduce_scatter_work_wait(self, tensor):
+        def comm_fn(tensor, group=None):
+            in_tensors = [tensor.clone() + i for i in range(group.size())]
+            out_tensor = torch.zeros_like(tensor)
+            work = dist.reduce_scatter(out_tensor, in_tensors, group=group, async_op=True)
+            return work, out_tensor
+
+        self._test_work_wait(tensor, comm_fn=comm_fn)
+
```

Comment:
there's no cpu test? Oh is it because we don't have `reduce_scatter` in gloo haha

Reply:
yep, no `reduce_scatter` on gloo. :)